dashboard/templates/blocks/header.ejs

```diff
@@ -44,7 +44,7 @@
           <% if(user) { %>
             <li class="nav-item dropdown">
               <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" id="userDropdown" >
-                <img src="https://cdn.discordapp.com/avatars/<%= user.id %>/<%= user.avatar %>.png?size=32" width="25" height="25" class="d-inline-block align-middle rounded-circle" alt="">
+                <img src="https://cdn.discordapp.com/avatars/<%= user.id %>/<%= user.avatar %>?size=32" width="25" height="25" class="d-inline-block align-middle rounded-circle" alt="">

```

Comment:
The `.png` is not necissary and already provided for deafult avatars. This also allows gif avatars

Reply:
Shouldn't this use [displayAvatarURL](https://discord.js.org/#/docs/main/stable/class/ClientUser?scrollTo=displayAvatarURL)? It supports default avatars just fine.

Also I removed the `.gif` from my avatar's URL, it made a static (non-animated) webp, so I don't know how you got gif avatars.

Reply:
Sorry messed up there. Using avatarURL is the correct way
https://cdn.discordapp.com/avatars/406538226258411524/f69afe619cfd78376354cb03c40651f3
also my avatar is here and it works